src/main/flight/mixer_tricopter.c

```diff
@@ -536,12 +537,26 @@ STATIC_UNIT_TESTED void tailTuneModeThrustTorque(thrustTorque_t *pTT, const bool
         }
         else
         {
+            if (IsDelayElapsed_ms(pTT->lastAdjTime_ms, 1000))
```

Comment:
Should split out the gyro condition from the if and have this in the else condition. So we won't increase the limit when sticks are not centered.

Could also increase the limit by `dT * 0.1` without checking the timeout. We might get results faster. This might work also but takes longer to get samples.